src/test/java/hudson/tasks/junit/pipeline/JUnitResultsStepTest.java

```diff
@@ -257,6 +302,9 @@ public void stageInParallel() throws Exception {
 
         j.setDefinition(new CpsFlowDefinition("stage('outer') {\n" +
                 "  node {\n" +
+                "    touch 'first-result.xml'\n" +
+                "    touch 'second-result.xml'\n" +
+                "    touch 'third-result.xml'\n" +
```

Comment:
Why is this sort of change necessary? If it was passing before #348, why would it fail now?

Reply:
in theory it is because files are copied to the workspace before the build start so without a touch they should be ignored by the parsing. but finally I added https://github.com/jenkinsci/junit-plugin/pull/384/files#diff-d80b11180304e55de1556e79fdbd96a795037aeca3c3817cc52aa5734e966b0eR122 which was something existing before #348 but hardcoded somewhere in the code https://github.com/jenkinsci/junit-plugin/pull/348/files#diff-d80b11180304e55de1556e79fdbd96a795037aeca3c3817cc52aa5734e966b0eL224

Reply:
Not sure I followed that. Maybe @timja did? https://github.com/jenkinsci/junit-plugin/pull/384/files#diff-d80b11180304e55de1556e79fdbd96a795037aeca3c3817cc52aa5734e966b0eR122 points to a blank line? Tip: if you link to blob URLs with hashes (not branch names) and line numbers/ranges, GH will render the code inline, which makes it a lot easier to follow discussions.

Reply:
Example of the URL format, in case it helps: `https://github.com/jenkinsci/junit-plugin/blob/b232c5f48051edc49661726cbd03c49c0aee2a67/src/main/java/hudson/tasks/junit/TestResult.java#L223-L224` rendered as https://github.com/jenkinsci/junit-plugin/blob/b232c5f48051edc49661726cbd03c49c0aee2a67/src/main/java/hudson/tasks/junit/TestResult.java#L223-L224